src/holofuel/pages/CreateRequest/CreateRequest.test.js

```diff
@@ -227,6 +229,57 @@ describe('CreateRequest', () => {
     expect(within(getByTestId('counterparty-nickname')).getByText('Loading')).toBeInTheDocument()
   })
 
+  it('renders a clickable list of recent counterparties', async () => {
+    const agent1 = {
+      id: 'fkljd',
+      nickname: 'Jo'
+    }
+
+    const agent2 = {
+      id: 'dskajln',
+      nickname: 'Bob'
+    }
+
+    const mocks = [
+      {
+        request: {
+          query: HolofuelCounterpartyQuery,
+          variables: { agentId: agent1.id }
+        },
+        result: {
+          data: { holofuelCounterparty: agent1 }
+        }
+      },
```

Comment:
Same as in CreateOffer test.. Do we need this mock for this test?

Reply:
Good catch, removed.